AdyenDropIn/Presentation/DropInNavigationController.swift

```diff
@@ -7,120 +7,40 @@
 @_spi(AdyenInternal) import Adyen
 import UIKit
 
-internal final class DropInNavigationController: UINavigationController, PreferredContentSizeConsumer, AdyenObserver {
-    
+internal final class DropInNavigationController: UINavigationController {
+
+    // MARK: - Initializers
+
     internal typealias CancelHandler = (Bool, PresentableComponent) -> Void
     
     private let cancelHandler: CancelHandler?
     
     internal let style: NavigationStyle
     
     internal lazy var keyboardObserver = KeyboardObserver()
-    
-    internal init(rootComponent: PresentableComponent, style: NavigationStyle, cancelHandler: @escaping CancelHandler) {
+
+    // MARK: - Initializers
+
+    internal init(
+        rootViewController: UIViewController,
```

Comment:
Do we need this `rootViewController` in init?
`DropInNavigationController` is still just a `UINavigationViewController` and one can just reassign `viewControllers` array later.

Reply:
Nope, actually this whole class will dissappear.